lua/telescope/actions/init.lua

```diff
@@ -272,6 +272,16 @@ function actions.close_pum(_)
   end
 end
 
+--- Copy current entry command to unnamed plus register
+---@param prompt_bufnr number: The prompt bufnr
+function actions.copy_command_to_reg(prompt_bufnr)
+    -- concat ':' with command name entry
+    local function set_reg(entry) a.fn.setreg('+', ":" .. entry.name, "c") end
```

Comment:
Is there a need for a closure here? I think it's cleaner to just write
```lua
a.fn.setreg('+', ":" .. entry.name, "c")
```
in line 281.